src/dotnet/APIView/APIViewWeb/Languages/CppLanguageService.cs

```diff
@@ -199,7 +200,7 @@ private static void BuildNodes(CodeFileTokensBuilder builder, List<NavigationIte
             {
                 // Filter namespace based on file name if any of the namespace matches file name pattern
                 // If no namespace matches file name then allow all namespaces to be part of review to avoid mandating file name convention
-                if ((!foundFilterNamespace || nameSpace.StartsWith(packageNamespace)) && !nameSpace.EndsWith(DetailsNamespacePostfix))
+                if ((!foundFilterNamespace || nameSpace.StartsWith(packageNamespace)) && !(nameSpace.EndsWith(DetailsNamespacePostfix) || nameSpace.EndsWith(InternalNamespacePostfix)))
```

Comment:
The internal namespace should stay in the APIView.  Although we do not expect customers to use it, we do have a bar that it cannot have breaking changes.

Reply:
Yes, please! Thanks for picking that up @RickWinter